lang_tests/obscure.bf

```diff
@@ -0,0 +1,8 @@
+[
+  VM:
+    stdout: H
+]
+
+[ From http://www.hevanet.com/cristofd/brainfuck/tests.b ]
```

Comment:
Do we have to add any licenses if we are taking code from elsewhere?

Reply:
This is a bit tricky because that site has no license. However, the code snippet is so small that I believe it comes under the exception for fair use of quotations and, as suggested by various people, we have explicitly mentioned where the quote came from.

Reply:
That's fair enough. Just wanted to be sure.